harvester/aws-container-definitions.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "name": "${NAME}-container",
```

Comment:
Die NAME is hier niet nodig toch? Aangezien dit toch al specifiek voor de harvester is.

Reply:
Ja, klopt. Al doende een beetje overenigneerd. Als we in logConfiguration.options.awslogs-group de naam zouden opnemen, dan draaien harvester en service onder precies dezelfde container definition, maar laten we ze even apart houden ten gunste van flexibiliteit.